packages/workspace/test/common/nacl_file_store.ts

```diff
@@ -335,6 +335,7 @@ export const mockDirStore = (
     ),
     getFullPath: mockFunction<DirectoryStore<string>['getFullPath']>().mockImplementation(filename => filename),
     isPathIncluded: mockFunction<DirectoryStore<string>['isPathIncluded']>().mockReturnValue(true),
+    exists: mockFunction<DirectoryStore<string>['exists']>().mockResolvedValue(false),
```

Comment:
this is not a correct response most of the time
should be
```suggestion
    exists: mockFunction<DirectoryStore<string>['exists']>().mockImplementation(async filename => naclFiles.has(filename)),
```
or something like that.
right?